utils/hyper-body/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "hyper-body"
+version = "0.1.0"
```

Comment:
Where do we use this new package?

Reply:
openapi, kube-forward, kube-proxy and supportability